games-util/lutris/lutris-0.4.23.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+PYTHON_REQ_USE="sqlite,threads"
+
+inherit distutils-r1 gnome2-utils python-r1 xdg-utils
```

Comment:
Please replace `gnome2-utils` and `xdg-utils` with `xdg` eclass.

Reply:
This should allow EAPI to be bumped to 7, too. Previous lutris versions worked without python-r1, so you can most likely drop that too, and delete those useless phases below.